tests/data/test_types.py

```diff
@@ -27,7 +26,8 @@
 LENGTHS = [0, 1, 2, 10]
 
 
-def _check_1d_shape(fn: Callable[[np.ndarray], Any], length: float, expected_msg: str):
+def _check_1d_shape(fn: Callable[[np.ndarray], Any], length: int, expected_msg: str):
+    assert isinstance(length, int)
```

Comment:
Because this was a float before (incorrectly), and I just replaced the type. I added an assertion to see if my assumption of the true type was correct, and forgot to remove it.